include/SFML/Graphics/Font.hpp

```diff
@@ -238,13 +239,7 @@ class SFML_GRAPHICS_API Font
     ////////////////////////////////////////////////////////////
     /// \brief Retrieve the texture containing the loaded glyphs of a certain size
     ///
-    /// The contents of the returned texture changes as more glyphs
-    /// are requested, thus it is not very relevant. It is mainly
-    /// used internally by sf::Text.
-    ///
-    /// \param characterSize Reference character size
-    ///
-    /// \return Texture containing the glyphs of the requested size
+    /// \deprecated Use getGlyph(Uint32 codePoint, unsigned int characterSize, bool bold).texture instead.
```

Comment:
There's no need to "undocument" a function just because it's being marked as deprecated.